From f3d7b69ee29a524d6156d330926a2eb4ba3e258f Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Fri, 10 Feb 2012 13:26:21 +0800
Subject: [PATCH 1139/2463] ENGR00174062 CCM: change clock enable_count to
 usecount

change clock debugfs sys attr 'enable_count' to 'usecount'
to align with some power debug tool used

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 arch/arm/plat-mxc/clock.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-mxc/clock.c b/arch/arm/plat-mxc/clock.c
index e192357..4b92479 100755
--- a/arch/arm/plat-mxc/clock.c
+++ b/arch/arm/plat-mxc/clock.c
@@ -4,7 +4,7 @@
  * Copyright (C) 2004 - 2005 Nokia corporation
  * Written by Tuukka Tikkanen <tuukka.tikkanen@elektrobit.com>
  * Modified for omap shared clock framework by Tony Lindgren <tony@atomide.com>
- * Copyright 2007-2011 Freescale Semiconductor, Inc.
+ * Copyright 2007-2012 Freescale Semiconductor, Inc.
  * Copyright 2008 Juergen Beisert, kernel@pengutronix.de
  *
  * This program is free software; you can redistribute it and/or
@@ -362,7 +362,7 @@ static int clk_debug_register_one(struct clk *clk)
 
 	clk->dentry = d;
 
-	d = debugfs_create_u8("enable_count", S_IRUGO, clk->dentry,
+	d = debugfs_create_u8("usecount", S_IRUGO, clk->dentry,
 			(u8 *)&clk->usecount);
 	if (!d) {
 		err = -ENOMEM;
-- 
1.7.7.4

